x-pack/plugins/translations/translations/zh-CN.json

```diff
@@ -7611,75 +7611,41 @@
     "xpack.security.management.editRole.setPrivilegesToKibanaSpacesDescription": "设置 Elasticsearch 数据的权限并控制对 Kibana 空间的访问权限。",
     "xpack.security.management.editRole.updateRoleText": "更新角色",
     "xpack.security.management.editRole.viewingRoleTitle": "正在查看角色",
-    "xpack.security.management.editRoles.deleteRoleButton.cancelButtonLabel": "不，不删除",
```

Comment:
This is a simple rename from `.editRoles.` to `.editRole.`, to maintain consistency.

The other deletions below are removing entries that no longer exist on the space/role management screens as a result of the rewrite. The linter fails the build if these exist.